boot/bootutil/include/bootutil/image.h

```diff
@@ -17,6 +17,10 @@
  * under the License.
  */
 
+/*
+ * Modifications are Copyright  � 2019 Arm Limited.
```

Comment:
I think we should figure out how contributors should be inserting copyright statements. Rather than these kind of Ad-hoc declarations, what about the initial copyright lines, similar to the Linaro one in `boot/bootutil/src/caps.c`? The question that remains there is what do we put up there for the files that don't already have a copyright. I would guess that Runtime/Apache's lack of copyright statements will be a minority position for contributors. Perhaps we put in a "Modifications are Copyright" line at the top for those files, but it then gets funny as more and more of the work is done by these various parties, and they are no longer just modifications.

Another option would be to put either an Apache copyright, or a Runtime copyright at the tops of the files, depending on what is appropriate.

Reply:
The best would be to put similar copyright statement in all files in the repo like in `boot/bootutil/src/caps.c`. Then it would be straightforward to extend that list.

Can it be done with those files (retroactively), which currently lack of any copyright?